activejob/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Rails 7.1.0.beta1 (September 13, 2023) ##
 
+*   Clarify the backoff strategy for the recommended `:wait` option when retrying jobs
```

Comment:
`7.1.0.beta1` has already been released, this should go above it (with a newline in between)

Reply:
okay, I think I finally got it right. Thanks for the review!